src/DotnetKubernetesClient/IKubernetesClient.cs

```diff
@@ -181,6 +181,7 @@ Task Delete<TResource>(string name, string? @namespace = null)
         /// If the namespace is omitted, all resources on the cluster are watched.
         /// </param>
         /// <param name="cancellationToken">Cancellation-Token.</param>
+        /// <param name="labelSelector">A string, representing an optional label selector for filtering watched objects.</param>
```

Comment:
Not to be picky, but here, the documentation is not quite correct.
```suggestion
        /// <param name="labelSelector">A list of label selectors to apply to the watcher.</param>
```

Reply:
Oh, oops; not only that, but the param name is incorrect... it should be `labelSelectors`, to match `List<TResource`

Reply:
Fixed.